content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 5
+description: Automatic instrumentation for opentelemetry-swift
+---
+
+OpenTelemetry-Swift provides several instrumentation libraries.
+
+## `SDKResourceExtension`
+
+`SDKResourceExtension` provides details about the device as a Resource.
+
+### Usage
+
+Use `DefaultResource.get()` to generate an all-in-one resource object. This
+resource can be added to a `TracerProvider` or `MetricProvider`.
+
+```swift
+OpenTelemetry.registerMeterProvider(meterProvider: MeterProviderBuilder()
```

Comment:
Minor nit, but I'd recommend using `tracerProvider` since it's more stable, but I also don't really know the intended audience that well. If iOS folks are more like to use metrics then maybe it's better to have that here.